javascript/modules/client/README.md

```diff
@@ -1663,6 +1758,74 @@ const configAction = client.encoding.updatePluginSettingsAction(
 
 ## Action decoders
 
+### Decode action grant permission
+
+```ts
+import { Client, Context, IPermissionParams, Permissions } from "@aragon/sdk-client";
+
+const context: Context = new Context(params);
+const client: Client = new Client(context);
+
+const data: Uint8Array =  new Uint8Array([12, ..., 56])
+
+const grantParams = client.decoding.grantAction(
+  data
+);
+console.log(grantParams);
+/*
+{
+  who: "0x1234567890...",
+  where: "0x1234567890...",
+  permission: "UPGRADE_PERMISSION",
```

Comment:
Could we include the `permissionId` as well? Could we also mention that `permission` may be null if the permission cannot be identified?